packages/swarm-styles/lib/components/toggle.css

```diff
@@ -21,6 +21,10 @@
     background-color: #FAFAFA;
 }
 
+[data-swarm-width="grow"] {
```

Comment:
This duplicates the [styles declared in button](https://github.com/meetup/swarm-ui/pull/76/files?file-filters%5B%5D=.css&file-filters%5B%5D=.jsx&file-filters%5B%5D=.mdx&file-filters%5B%5D=.png#diff-03a2107e1292e8924ad8cb2234574206R32) since they're all minified to the same file. For the shared component style we could defined a shared.css in the component dir.